lib/hbase/hbase-client.js

```diff
@@ -244,14 +244,33 @@ HbaseClient.prototype.getAccountPayments = function (options, callback) {
   var startRow = options.account + '|' + utils.formatTime(options.start);
   var endRow   = options.account + '|' + utils.formatTime(options.end);
 
-  this.getScan({
-    table      : table,
-    startRow   : startRow,
-    stopRow    : endRow,
-    limit      : options.limit,
-    descending : options.descending
-  }, function (err, rows) {
-    callback(err, formatPayments(rows || []));
+  if(options.currency) {
+    options.currency= options.currency.toUpperCase();
+  }
+
+  if(options.type) {
+    options.type= options.type.toLowerCase();
+  }
+
+  var maybeFilters =
+  [{ qualifier: 'currency', value: options.currency, family: 'f', comparator: '=' },
+   { qualifier: options.type, value: options.account, family: 'f', comparator: '=' }];
```

Comment:
type should be 'sent' or 'received' with logic here to translate it into the appropriate filters for 'source' and 'destination' columns